backend/tracim_backend/lib/search/elasticsearch_search/elasticsearch_search.py

```diff
@@ -681,10 +682,8 @@ def search_user(
                 "range", newest_authored_content_date=newest_authored_content_date_range
             )
 
-        if size:
-            search = search.extra(size=size)
-        if page_nb:
-            search = search.extra(from_=self.offset_from_pagination(size, page_nb))
+        if size and page_nb:
```

Comment:
I don't understood this code, you should keep possibility to set size (and not just offset) and allow 0 size too (for facet only case)

Reply:
? setting `size=0/None` had no effect, even with the previous code. So this means it wasn't possible to only get facets and not results. By the way those two values (`page_nb` and `size`) are enforced to be strictly positive by the schema and defaults to `1`/`10`. So I'm even wondering whether we should care about false values here…

Reply:
According to content search specification (last version), we should now allow 0 for size, as this permit (that's a elasticsearch feature) to have facet only search (this work with content search so i see no reason this should not work here). Also, as far i understood your fix, you remove the ability to set size, which is not something we want.

see `test_api___elasticsearch_search_ok__wildcard__facet_only` for example.

Reply:
Corrected, I wasn't aware of the `size=0` case.